### ISSUE TITLE
code error!

### ISSUE THREAD

**Issue by xiyufeng2** [Jamie725/Multimodal-Object-Detection-via-Probabilistic-Ensembling#6]
When I run 'demo_train_middle_fusion.py', the input has the dimension of 3, which lead to the error 'The size of tensor a (3) must match the size of tensor b (6) at non-singleton dimension 0'.
Maybe there is something wrong with your code.

**Comment by Jamie725** [comment]
It's not a code error. 
Please check if you install detectron 2 correctly. I changed the read_image here: https://github.com/Jamie725/Multimodal-Object-Detection-via-Probabilistic-Ensembling/blob/main/detectron2/data/detection_utils.py#L38
It should be able to recognize 6 channel input as long as you set the format to be "BGRTTT"

**Comment by Jamie725** [comment]
It's not a code error. 
Please check if you install detectron 2 correctly. I changed the read_image here: https://github.com/Jamie725/Multimodal-Object-Detection-via-Probabilistic-Ensembling/blob/main/detectron2/data/detection_utils.py#L38
It should be able to recognize 6 channel input as long as you set the format to be "BGRTTT"

**Comment by xiyufeng2** [comment]
I followed the command :'python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'' to biuld detectron2, whose name conflicts with an existing folder ‘detectron2’. So I changed  the name of an existing folder detectron2 to be Detectron because there is no setup.py file in it. Finally, I merged these two folders。

I'll try your updated files ‘detection_utils.py#L38)’ to see if I can run successfully.  Thanks for your reply!
